python/mxnet/libinfo.py

```diff
@@ -73,6 +73,11 @@ def find_lib_path(prefix='libmxnet'):
                            'List of candidates:\n' + str('\n'.join(dll_path)))
     if os.name == 'nt':
         os.environ['PATH'] = os.environ['PATH'] + ';' + os.path.dirname(lib_path[0])
+        if sys.version_info >= (3, 8):
+            if 'CUDA_PATH' not in os.environ:
+                raise RuntimeError('Cannot find the env CUDA_PATH.Please set CUDA_PATH env with cuda path')
```

Comment:
Thus mxnet won't work on Windows as long as users don't set up this environment variable? Or is it provided by default by cuda?

Reply:
cuda default provided.